ext/standard/tests/array/krsort_variation9.phpt

```diff
@@ -82,22 +82,22 @@ array(5) {
 - With default sort flag -
 bool(true)
 array(3) {
+  ["c"]=>
+  string(5) "apple"
   ["a"]=>
   string(6) "orange"
   [0]=>
   string(6) "banana"
-  ["c"]=>
-  string(5) "apple"
 }
 - Sort flag = SORT_REGULAR -
 bool(true)
 array(3) {
+  ["c"]=>
+  string(5) "apple"
   ["a"]=>
   string(6) "orange"
   [0]=>
   string(6) "banana"
-  ["c"]=>
-  string(5) "apple"
 }
```

Comment:
This looks wrong; the old result was correct. Same for the other `krsort()` test below.

Reply:
The order of keys by according to the old result: `["a", 0, "c"]`
Let's check it with a spaceship operator:
`print_r(["a" <=> 0, 0 <=> "c"]);` gives the following output:
```
Array
(
    [0] => 1
    [1] => -1
)
```
Which by definition means that it's not sorted and therefore the old result is incorrect.

Let's do the same with the new result: `["c", "a", 0]`
`print_r(["c" <=> "a", "a" <=> 0]);` gives us the following:
```
Array
(
    [0] => 1
    [1] => 1
)
```
Which means that it's sorted and actually correct.

Reply:
Of course, you're right! I was confused; sorry for the noise.